lib/compress.js

```diff
@@ -190,6 +190,14 @@ merge(Compressor.prototype, {
         return false;
     };
 
+    function can_be_evicted_from_block(node) {
```

Comment:
Like `creates_block_scope`, we could extract this into an `AST_Node.is_block_scoped` method? Fine as-is though.